src/net.cpp

```diff
@@ -1760,12 +1760,12 @@ int CConnman::GetExtraOutboundCount()
     {
         LOCK(cs_vNodes);
         for (const CNode* pnode : vNodes) {
-            if (pnode->fSuccessfullyConnected && !pnode->fDisconnect && pnode->IsOutboundOrBlockRelayConn()) {
+            if (pnode->fSuccessfullyConnected && !pnode->fDisconnect && pnode->IsFullOutboundConn()) {
                 ++nOutbound;
             }
         }
     }
-    return std::max(nOutbound - m_max_outbound_full_relay - m_max_outbound_block_relay, 0);
+    return std::max(nOutbound - m_max_outbound_full_relay, 0);
```

Comment:
Perhaps it's a good opportunity to refactor this function:
- rename `GetExtraOutboundCount` -> `GetExtraFullOutboundCount`
- rename `int nOutbound` -> `uint32_t full_outbound_peers`

Reply:
Fixed.